README.md

```diff
@@ -66,10 +66,17 @@ For instance, when we released v1.22.0, the latest Kubernetes version was v1.20.
 
 The Jaeger Operator *might* work on versions outside of the given range, but when opening new issues, please make sure to test your scenario on a supported version.
 
-| Jaeger Operator | Kubernetes           |
-|-----------------|----------------------|
-| v1.23           | v1.19+               |
-| v1.22           | v1.18 to v1.20       |
+| Jaeger Operator | Kubernetes           | Strimzi Operator   |
```

Comment:
Reading the doc in a broader context, I believe this table should be moved one section up, as it includes versions from two sections.